cluster/manifests/skipper/deployment.yaml

```diff
@@ -39,6 +39,8 @@ metadata:
 spec:
 {{ if index . "replicas" }}
   replicas: {{ .replicas }}
+{{ else if eq .Cluster.ConfigItems.skipper_ingress_test_single_pod "true" }}
```

Comment:
Hm, I think it would be cleaner to do
```yaml
{{ if eq .Cluster.ConfigItems.skipper_ingress_test_single_pod "true" }}
{{ template "skipper-ingress" dict
  "name" "skipper-ingress"
  "internal_version" $internal_version
  "replicas": 0
  "Cluster" .Cluster
  "Values" .Values
}}
{{ else }}
{{ template "skipper-ingress" dict
  "name" "skipper-ingress"
  "internal_version" $internal_version
  "Cluster" .Cluster
  "Values" .Values
}}
{{ end }}
```

Reply:
Also skipper_ingress_test_single_pod implis test and single, but it is single canary pod which in future might be several pods.
Maybe it should be just `skipper_ingress_replicas` that is empty by default:
```yaml
{{ if ne .Cluster.ConfigItems.skipper_ingress_replicas "" }}
{{ template "skipper-ingress" dict
  "name" "skipper-ingress"
  "internal_version" $internal_version
  "replicas": .Cluster.ConfigItems.skipper_ingress_replicas
  "Cluster" .Cluster
  "Values" .Values
}}
{{ else }}
{{ template "skipper-ingress" dict
  "name" "skipper-ingress"
  "internal_version" $internal_version
  "Cluster" .Cluster
  "Values" .Values
}}
{{ end }}
```